conftest.py

```
@@ -0,0 +1,281 @@
+            pass
+
+        # Also patch OpenAI client to use verify=False for httpx
+        try:
+            import httpx
+            import openai
+
+            _original_openai_init = openai.OpenAI.__init__
+
+            def _patched_openai_init(self, *args, **kwargs):
+                if "http_client" not in kwargs:
+                    kwargs["http_client"] = httpx.Client(verify=False)
```

Ruff (0.14.13)

Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)